ui/packages/consul-ui/app/adapters/binding-rule.js

```diff
@@ -0,0 +1,28 @@
+import Adapter from './application';
+
+export default class BindingRuleAdapter extends Adapter {
+  requestForQuery(request, { dc, ns, authmethod, index, id }) {
+    return request`
+      GET /v1/acl/binding-rules?${{ dc, authmethod }}
+
+      ${{
+        ...this.formatNspace(ns),
+        index,
+      }}
+    `;
+  }
+
+  requestForQueryRecord(request, { dc, ns, index, id }) {
```

Comment:
@johncowen We're not using this endpoint. Should I delete it? All the adapter tests are for this endpoint so that would have to go too. 

If it stays, I would need to do mock-data for it.

Reply:
Yeah if we never need to request a single binding rule on its own, you can just delete this if you like (plus tests), we can just add it back later if/when we need it.